apps/prairielearn/src/components/CourseRequestsTable.html.ts

```diff
@@ -125,7 +125,7 @@ export function CourseRequestsTable({
                     ${row.jobs.length > 0
                       ? html`
                           <button
-                            class="show-hide-btn btn btn-secondary btn-sm collapsed btn-xs text-nowrap"
```

Comment:
Was this removal of `btn` intentional? Also, this uses `btn-sm` and `btn-xs` at the same time, aren't they contradictory?

Reply:
This was not intentional, I think this somehow got caught up in fixes of duplicate `btn` classes elsewhere. Good catch.

Applying `btn-sm` and `btn-xs` is definitely bad. I'll fix that too.

Reply:
Addressed in 597175b.